src/absil/ilreflect.fs

```diff
@@ -324,25 +324,7 @@ type cenv =
 // [ns]            ,name -> ns+name
 // [ns;typeA;typeB],name -> ns+typeA+typeB+name
 let convTypeRefAux (cenv:cenv) (tref:ILTypeRef) = 
-
-    // If an inner nested type's name contains a space, the proper encoding is "\+" on both sides - otherwise,
-    // we use "+"
```

Comment:
As far as I can tell from my testing, this comment is just *wrong* -- I can't find any scenarios where the + is escaped. Indeed, the incorrect escaping needed to be removed in order to get my tests passing. All current tests continue to work fine.

This is now simplified and exactly matches https://github.com/Microsoft/visualfsharp/blob/fsharp4/src/absil/il.fs#L682